android-social/alipay/build.gradle.kts

```diff
@@ -0,0 +1,5 @@
+group = "io.logto.sdk"
+version = libs.versions.logtoSdk.get()
+
+configurations.create("default")
+artifacts.add("default", file("alipaySdk-15.7.9-20200727142846.aar"))
```

Comment:
we should not put it in source code per discussion in swift sdk

Reply:
you can leverage https://github.com/logto-io/social-sdks if needed

Reply:
We need to publish it if we put this into another repo, it's very inconvenient to share the libs between multi repos in the development process. This is a very common practice in Gradle.

Reply:
updated